ClosedXML/Excel/Comments/XLComment.cs

```diff
@@ -6,12 +6,28 @@ internal class XLComment : XLFormattedText<IXLComment>, IXLComment
     {
         private XLCell _cell;
 
+        public XLComment(XLCell cell)
+            : this(cell, XLFont.DefaultCommentFont)
+        {
+        }
+
         public XLComment(XLCell cell, IXLFontBase defaultFont)
             : base(defaultFont)
         {
             Initialize(cell);
         }
 
+        public XLComment(XLCell cell, int shapeId)
+            : this(cell, XLFont.DefaultCommentFont, shapeId)
+        {
+        }
+
+        public XLComment(XLCell cell, IXLFontBase defaultFont, int shapeId)
+            : base(defaultFont)
+        {
+            Initialize(cell, shapeId);
+        }
```

Comment:
I would move from having 6 constructors toward 2-3 with optional parameters.

How about this?
```
public XLComment(XLCell cell, IXLFontBase defaultFont = XLFont.DefaultCommentFont, int? shapeId = null)
: base(defaultFont ?? XLFont.DefaultCommentFont)
{
  if (cell == null) throw new ArgumentNullException(nameof(cell));
  shapeId = shapeId ?? cell.Worksheet.Workbook.ShapeIdManager.GetNext();
  Initialize(cell, shapeId.Value);
}
```

Ideally, only one constructor should call `base()`, all the others call `this()` with different sets of parameters. But this is not the case as the base class already has different constructors.

Reply:
Ok, good point.

Reply:
This has been fixed.